Reference/Routing/routing-properties-v8.md

```diff
@@ -31,4 +31,4 @@ The URL alias remains in the browser address bar as a 'mask' over the real URL.
 
 ## Filtering
 
-[See Filtering Property Conventions](../Querying/IPublishedContent/Collections/index-v8.md#filtering-conventions)
\ No newline at end of file
+[See Filtering Property Conventions](../Querying/IPublishedContent/Collections/)
```

Comment:
Hey @Ambertvu 
This is actually not a link to the page. it should specify the file on the link.

```suggestion
[See Filtering Property Conventions](../Querying/IPublishedContent/Collections/index-v8.1.md#filtering-conventions)
```